tests/feature/test_background.py

```diff
@@ -1,39 +1,42 @@
 """Test feature background."""
-import re
-
-import pytest
 
-from pytest_bdd import given, parsers, scenario, then
+import textwrap
 
 
-def test_background_basic(request):
-    """Test feature background."""
-
-    @scenario("background.feature", "Basic usage")
-    def test():
-        pass
+FEATURE = """\
+Feature: Background support
 
-    test(request)
+    Background:
+        Given foo has a value "bar"
+        And a background step with multiple lines:
+            one
+            two
 
 
-def test_background_check_order(request):
-    """Test feature background to ensure that backound steps are executed first."""
+    Scenario: Basic usage
+        Then foo should have value "bar"
 
-    @scenario("background.feature", "Background steps are executed first")
-    def test():
-        pass
+    Scenario: Background steps are executed first
+        Given foo has no value "bar"
+        And foo has a value "dummy"
 
-    test(request)
+        Then foo should have value "dummy"
+        And foo should not have value "bar"
+"""
 
+STEPS = """\
+import re
+import pytest
+from pytest_bdd import given, then, parsers
 
 @pytest.fixture
 def foo():
     return {}
 
 
-@given(parsers.re(r"a background step with multiple lines:\n(?P<data>.+)", flags=re.DOTALL))
+@given(parsers.re(r"a background step with multiple lines:\\n(?P<data>.+)", flags=re.DOTALL))
```

Comment:
you can use `r"""` at line 27, so that you don't have to do weird string escapes here.